server/routes/objects.js

```diff
@@ -246,52 +239,41 @@ router.post('/:url', async function (req, res) {
         let tempName = parts[0];
         let inProgress = req.query.inProgress;
         let socket_id = req.query.socket_id;
-        let fileId = uuidv4();
         let pngFileName = tempName + '.png';
         let tempDirPath = path.resolve(__dirname, '../temp');
 
-        await map_file_type(user, fileId, bucketName, tempName, parts[1]);
-        // let fileInfo = await file_stats(bucketName, tempName);
-        // let fileId = fileInfo[0].file_unique_id;
+        await map_file_type(bucketName, tempName, parts[1]);
+
         if (
           files.file[0].mimetype === 'image/jpeg' ||
           files.file[0].mimetype === 'image/png'
         ) {
+          sockets[socket_id].disconnect();
+          removeSocket(socket_id);
           minioClient.fPutObject(
             bucketName,
-            'hv/' + user + '/thumbnail/' + fileName + fileId,
+            'hv/' + user + '/thumbnail/' + fileName,
             filePath,
             async (err, objInfo) => {
               if (err) {
+                log.error('Failed to Upload jpeg/png Thumbnail');
                 return res.status(400).json({ error: 'Failed to upload' });
               }
-              // await file_uploaded(user, bucketName, tempName, parts[1]);
+              await file_uploaded(bucketName, tempName, parts[1]);
               res
```

Comment:
The error handling for directory deletion should log the error using the logger instead of console.error. This ensures consistent logging throughout the application.
```diff
-                    console.error(
+                    log.error(
```
*This comment was auto-generated by **Code Bot** to provide suggestions and improvements.*